apps/src/netsim/DashboardUser.js

```diff
@@ -91,7 +91,9 @@ DashboardUser.getCurrentUser = function () {
         DashboardUser.currentUser_.initialize_(data);
       },
       error: function (/*jqXHR, textStatus, errorThrown*/) {
-        throw new Error("Unable to retrieve current user info.");
+        DashboardUser.currentUser_.initialize_({
```

Comment:
Okay, good catch on both counts.  RubyMine actually complains about private access here, AND I changed the function signature at some point without fixing the comment up.